packages/plugins/type-designer/src/components/elements-type-container/content-tabs.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+  import Card from "@smui/card";
+  import Paper from "@smui/paper";
+  import Button from "@smui/button";
+  import type { DataTypeTemplates } from "./data-type-templates";
+  import { createNewItem } from "./create-new-item";
+  import { TypeDesignerService } from "../type-designer-service";
+    import { ELEMENT_NAMES, type DataTypeTemplatesQueries } from "@oscd-plugins/core";
+
+  export let scdQueries: DataTypeTemplatesQueries;
+
+  type Column = {
+    name: string;
+    visible: boolean;
+    items: DataTypeTemplates[];
+  };
+
+  const service = new TypeDesignerService(scdQueries);
+  const logicalDevices = service.findAllLogicalDeviceElements();
+  const bays = service.findAllBayElements();
+  const ieds = service.findAllIEDElements();
+  const voltageLevels = service.findAllVoltageLevelElements();
+  const substations = service.findAllSubstationElements();
+  console.log("found: ", logicalDevices);
```

Comment:
do we need this still?

Reply:
Somehow you reviewed an non-existing file :) This is no longer in the codebase